recipe/meta.yaml

```
@@ -40,11 +34,12 @@ requirements:
     - streamlit-camera-input-live >=0.2.0
     - streamlit-vertical-slider >=1.0.2
```

```suggestion
    - streamlit-vertical-slider >=2.5.5
```
see https://github.com/arnaudmiribel/streamlit-extras/blame/v0.4.0/pyproject.toml#L27